plugins/node/instrumentation-amqplib/src/amqplib.ts

```diff
@@ -74,6 +74,8 @@ import {
 } from './utils';
 import { VERSION } from './version';
 
+const supportedVersions = ['>=0.5.5 <1'];
```

Comment:
I love having this pulled out at the top, instead of hardcoded multiple times throughout. I don't know if it's necessary here because you're already going to be dealing with a lot of merge conflicts, but it might be nice to do this across all instrumentations too, so versions are easier to find and update instead of digging through the code.

Reply:
I think so too, support doing this in a follow-up PR